plugins/database/plugin.go

```diff
@@ -61,45 +61,57 @@ func createStore() {
 func configure(_ *node.Plugin) {
 	// assure that the store is initialized
 	store := Store()
+	configureHealthStore(store)
 
-	err := checkDatabaseVersion(store.WithRealm([]byte{prefix.DBPrefixDatabaseVersion}))
-	if errors.Is(err, ErrDBVersionIncompatible) {
-		log.Panicf("The database scheme was updated. Please delete the database folder.\n%s", err)
-	}
-	if err != nil {
+	if err := checkDatabaseVersion(store.WithRealm([]byte{prefix.DBPrefixDatabaseVersion})); err != nil {
+		if errors.Is(err, ErrDBVersionIncompatible) {
+			log.Panicf("The database scheme was updated. Please delete the database folder.\n%s", err)
```

Comment:
Above we use `Fatal` and here `Panic` is there a particular reason for that?

Reply:
Also the has a `\n`

Reply:
Made it use Fatal everywhere